Misc/NEWS.d/next/Core and Builtins/2018-12-22-22-19-51.bpo-35560.9vMWSP.rst

```diff
@@ -0,0 +1,3 @@
+Removed an incorrect assert statement expecting `min_width`  >= 0 in
```

Comment:
`min_width` and `_PyUnicode_InsertThousandsGrouping` are implementation details. Describe the issue in terms of Python.